build/dependencies.props

```diff
@@ -4,6 +4,9 @@
     <!-- MicrosoftNETCoreApp21PackageVersion is assigned at the bottom so it can automatically pick up MicrosoftNETCoreAppPackageVersion in an orchestrated build. -->
     <MicrosoftNETCoreAppPackageVersion>2.1.22</MicrosoftNETCoreAppPackageVersion>
     <MicrosoftNETCoreDotNetAppHostPackageVersion>2.1.22</MicrosoftNETCoreDotNetAppHostPackageVersion>
+    <MicrosoftExtensionsCachingAbstractionsPackageVersion>2.1.2</MicrosoftExtensionsCachingAbstractionsPackageVersion>
```

Comment:
@dougbu The `Microsoft.Extension.Caching.Abstractions` and `Microsoft.Extensions.Caching.Memory` packages are jumping from 2.1.2 to 2.1.23 in this patch (nuget.org has no intervening versions). Is this intentional? Shouldn't those be 2.1.3.

Reply:
Gaps are expected because Extensions doesn't build every assembly in every servicing release. Very likely these assemblies weren't updated since 2.1.2. They're definitely in 2.1.23. See https://github.com/dotnet/extensions/blob/dc1dab7c23212e095e280d202fa2151daa01e393/eng/PatchConfig.props#L14-L19

After the release, we'll probably need to repin all three of these packages at the versions produced in this release. Right @mmitche❔

Reply:
Yes correct. Just making sure that versioning is correct.